### PR TITLE
Update staff list again

### PR DESCRIPTION
**Pull request by CesiumCs**
Should be up to date and alphabetized, waiting for input from #staff

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,52 +33,59 @@ New features should also be reported as [issues]. Please try to describe the fea
...
-* sakuya
-* [vitor is]       
-* xpielordx
-* ForkyFork
+
 * Babbaj
+* BeeVa
+* [Berrely]
+* Ezioblade12
+* FakeAccount1231
```
Comment (l1ving) [inline README.md:65]: wh

**Review by l1ving — COMMENTED** [review]
I don't know all the people on staff so wait for @zPeanut or @leijurv to review

File: README.md (hunk 1 above)
Comment (CesiumCs) [reply README.md:65]: I just went through everyone with the roles

**Comment by zPeanut** [comment]
lgtm

**Pull request merged**